doc/eggex.md

```diff
@@ -311,6 +311,21 @@ Flags and translation preferences together:
 
     / digit+ ; ignorecase ; python /  # could translate to (?i)\d+
 
+The following flags are currently supported:
```

Comment:
I would say "In Oils, the following flags ..."

There is this idea that people can implement Eggex on top of other regex engines ... not sure how appealing it is, but it is supposed to be more abstract